ProcessMaker/Listeners/UserSession.php

```diff
@@ -51,7 +51,7 @@ public function handle(object $event): void
             $user->sessions()
                 ->where('is_active', true)
                 ->where('ip_address', $ip)
-                ->update(['expired_date' => now()]);
+                ->update(['expired_date' => now()->toDateTimeString()]);
```

Comment:
I still got the error:

![image](https://github.com/ProcessMaker/processmaker/assets/8028650/69043988-86e0-448c-99ce-32707220e696)

Seems the $sql generated by this line is:

```
update `user_sessions` set `expired_date` = ?,
    `user_sessions`.`updated_at` = ?
where
    `user_sessions`.`user_id` = ?
    and `user_sessions`.`user_id` is not null
    and `is_active` = ?
    and (`device_name` != ? or `device_type` != ? or `device_platform` != ?)
```
And the binding values are something like:
```
[
0 => '2024-01-05 21:07:14',
  1 => '2024-01-05 21:08:06',
  2 => 4,
  3 => true,
  4 => false,
  5 => 'desktop',
  6 => 'Ubuntu',
]
```

Reply:
Seems that the parameter with index 4 is the problem (4=>false) that corresponds to the device name is false instead of a valid device name like "Machintosh", etc.

Reply:
Seems that the user agent coming from Firefox (Windows/Ubuntu) doesn't have this information.
However, I added an extra validation for the device name.